yarn-project/world-state/src/world-state-db/index.ts

```diff
@@ -16,6 +16,13 @@ export enum MerkleTreeId {
 
 export type IndexedMerkleTreeId = MerkleTreeId.NULLIFIER_TREE;
 
+/**
+ * The nullifier tree must be pre filled with the number of leaves that are added by one rollup.
+ * The tree must be initially padded as the pre-populated 0 index prevents efficient subtree insertion.
+ * Padding with some values solves this issue.
+ */
+export const INITIAL_NULLIFIER_TREE_SIZE = 8;
```

Comment:
Just to be sure. Is this taking us down a road of fixed size rollups?

Reply:
I don't think it will have that effect long term, it will just need to be the size of a nullifier tree sub tree insertion